#3 Create orders table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE orders(
+o_id INT,
+order_Date date,

```

Comment:
Probably in MySQL, you can find a type that will store both date and time in one column.